src/views/ManageList.jsx

```diff
@@ -1,7 +1,69 @@
-export function ManageList() {
+import { useState } from 'react';
+import { addItem } from '../api/firebase';
+
+export function ManageList({ listPath }) {
+	const [itemName, setItemName] = useState('');
+	const [daysUntilNextPurchase, setDaysUntilNextPurchase] = useState(7);
+	const [message, setMessage] = useState('');
+
+	const handleSubmit = async (event) => {
+		event.preventDefault();
+		try {
+			await addItem(listPath, { itemName, daysUntilNextPurchase });
+			setMessage('Item was successfully saved to the database.');
+		} catch (error) {
+			setMessage('There was an error saving the item to the database.');
```

Comment:
When I first tried to add an item in the deploy preview, I got the error message, because I hadn't selected a list on the Home page yet (so there was no listPath set). Not a problem for now, but could be an edge case worth considering when y'all are revising your designs down the line!